src/material-experimental/mdc-slider/slider.scss

```diff
@@ -1,5 +1,7 @@
 @import '@material/slider/slider';
-@include core-styles;
+@import '../mdc-helpers/mdc-helpers';
+
+@include without-ripple($query: $mat-base-styles-query);
```

Comment:
Same comment to use `@use` so this is prefixed.

Reply:
@jelbourn do we care about guarding MDC styles with some Angular Material class, or are we ok with just emitting the MDC styles unguarded like this?

I think in some components we guarded it with the host selector, but it does increase the specificity of everything then, and adds more bytes

Reply:
Off the top of my head I don't know if it's strictly necessary- I can't see a way it would realistically cause a conflict

Reply:
Just wanted to ask since I know we did it in some other components. I agree that it doesn't really hurt anything here.

It may be more of an issue when done in a color / typograpy mixin, because then the styles could cause weird theming behavior on other non-Angular MDC components. Mixing Angular with non-Angular MDC components sounds like a pretty edge case though.